Source/NSMenuToolbarItem.m

```diff
@@ -0,0 +1,90 @@
+/* Implementation of class NSMenuToolbarItem
+   Copyright (C) 2024 Free Software Foundation, Inc.
+
+   By: Gregory John Casamento
+   Date: 03-05-2024
+
+   This file is part of the GNUstep Library.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import "AppKit/NSMenuToolbarItem.h"
+#import "AppKit/NSMenu.h"
+#import "AppKit/NSImage.h"
+
+#import "GNUstepGUI/GSTheme.h"
+
+@interface NSToolbarItem (Private)
+- (NSView *) _backView;
```

Comment:
You don‘t see to use this method. Is it a left over from a previous attempt implement this class? Maybe that other version was a better fit?

Reply:
This method is something I thought I would need but ended up not using.  I have removed it.